usr/share/doc/find-the-command/ftc.fish

```diff
@@ -50,8 +50,11 @@ function _cnf_asroot
     if test (id -u) -ne 0
         if $_cnf_force_su
             su -c "$argv"
-        else
+        else if command -v sudo > /dev/null 2>&1 && sudo -l > /dev/null 2>&1
+        then
             sudo $argv
+        else if command -v doas > /dev/null 2>&1 && doas -C /etc/doas.conf > /dev/null 2>&1
```

Comment:
Why check if the `/etc/doas.conf` file is valid. I would think it would be better to try and use it regardless. Otherwise you may skip running the command all together and the user has no idea why it didn't execute.